src/server/sv_client.cpp

```diff
@@ -1653,3 +1638,19 @@ void SV_ExecuteClientMessage( client_t *cl, msg_t *msg ) {
 //	}
 }
 
+int SV_ClientRate( client_t *client )
+{
+	int minRate = sv_minRate->integer;
+	int maxRate = sv_maxRate->integer;
+
+	// Special case for sv_maxRate 0: "unlimited" was hardcoded to 90000 in jk2ded
+	if ( !maxRate ) maxRate = 90000;
```

Comment:
As `sv_maxRate` is not a new cvar the default value of "90000" only affects new servers. Existing servers with "0" as value are currently affected by the original upper limit of "90000" on jk2ded. If we don't have this magic value servers are going to behave different after updating jk2mv and that's something we usually want to avoid. So the magic value has to stay for consistency.